README.md

```diff
@@ -30,7 +30,7 @@
 
 **Location:** GitHub Issues [(see all accepted proposals).](https://github.com/withastro/roadmap/issues)
 
-**What to Expect:** A proposal reaches this stage (aka "is accepted") during a meeting with Maintainers and TSC, following our existing [RFC Proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process.
+**What to Expect:** A proposal reaches this stage (aka "is accepted") during an [#rfc-meetings](RFC Meeting) with Maintainers and TSC, following our existing [RFC voting](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) process.
```

Comment:
```suggestion
**What to Expect:** A proposal reaches this stage (aka "is accepted") during an [RFC Meeting](#rfc-meetings) with Maintainers and TSC, following our existing [RFC voting](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) process.
```

Reply:
Going with what Matthew said yesterday in Community Call it refers to `>= L2`, but I agree we should be precise here.

Reply:
Updated, thanks!